frontend/src/components/common/Filter/common/entities.ts

```diff
@@ -79,46 +81,6 @@ export interface CollectionRow extends Categories, PublisherMetadataCategories {
   summaryCitation: string;
 }
 
-/**
```

Comment:
Timmy, my apologies, it looks like there were a couple of duplicated variables after I resolved rebase conflicts from my last PR (and my workspace didn't highlight as errors for some reason). I have removed them here.

Reply:
Ah didn't spot that either sorry 😆 !

I think it's probably because TypeScript allows declaration merging!

https://www.typescriptlang.org/docs/handbook/declaration-merging.html